proteobench/plotting/plot_quant.py

```diff
@@ -205,7 +205,7 @@ def plot_metric(
             width=700,
             height=700,
             xaxis=dict(
-                title="Mean absolute difference between measured and expected log2-transformed fold change",
+                title="Median absolute difference between measured and expected log2-transformed fold change",
```

Comment:
Shouldnt the title be changed based on what is plotted?

Reply:
Yes, and that's what's currently implemented on https://github.com/Proteobench/ProteoBench/tree/mean_median_plot

I'll make a new PR from that branch, then I think this one should be closed without merging